CHANGELOG.md

```diff
@@ -1,3 +1,83 @@
+# v0.40.0 Release - 09/26/2019
+
+*Note*: This release comes with a new config version `v1beta16`.
+        To upgrade your `skaffold.yaml`, use `skaffold fix`. If you don't upgrade, skaffold will auto-upgrade in memory as best it can, and print a warning message.
+        See [Skaffold Deprecation Policy](http://skaffold.dev/docs/references/deprecation/) for details on what beta means.
+
+
+New Features:
+
+* Add option to override kubecontext from `skaffold.yaml` [#2510](https://github.com/GoogleContainerTools/skaffold/pull/2510)
+* Support YAML anchors in skaffold.yaml (key must start with a dot) [#2836](https://github.com/GoogleContainerTools/skaffold/pull/2836)
+* Add file sync to Event and State APIs [#2978](https://github.com/GoogleContainerTools/skaffold/pull/2978)
+* Implement 'skaffold render' for kubectl deployer [#2943](https://github.com/GoogleContainerTools/skaffold/pull/2943)
```

Comment:
This is big!!  a lot of users were looking forward to this. We shd make this more noticeable. 
```suggestion
* A new [`skaffold render`](https://skaffold.dev/docs/references/cli/#skaffold-render) command which renders manifests for kubectl deployer [#2943](https://github.com/GoogleContainerTools/skaffold/pull/2943)
```

Reply:
done